module/VuFind/src/VuFind/ChannelProvider/Facets.php

```diff
@@ -236,26 +236,27 @@ protected function buildChannel(Results $results, $filter, $title,
             'title' => $title,
             'providerId' => $this->providerId,
         ];
+        $retVal['token'] = $this->getToken($filter, $title);
```

Comment:
Why have we changed this to always have a token set? The token is only supposed to be returned when we don't have time/resources to load the real data and we need to trigger an AJAX load on the client side. We can talk about the possibility of always including tokens if the UI can't work without it, but in that case, we'd need to be sure to do it consistently across channel providers, not just in this one.